File: app/src/main/java/vn/edu/utt/uttqlsv/view/activities/Splash.java
package vn.edu.utt.uttqlsv.view.activities;

import android.content.Intent;
import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import vn.edu.utt.uttqlsv.R;

public class Splash extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);

        Thread timer = new Thread() {
            @Override
            public void run() {
                try {
                    sleep(1200);
                } catch (InterruptedException e) {
                    e.printStackTrace();
                } finally {
                    Intent intent = new Intent(getApplicationContext(),Login.class);
                    startActivity(intent);
                }

                super.run();
            }
        };
        timer.start();
    }

    @Override
    protected void onPause() {
        super.onPause();
        finish();
    }
}

File: app/src/main/java/vn/edu/utt/uttqlsv/view/activities/Splash.kt
package vn.edu.utt.uttqlsv.view.activities

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import vn.edu.utt.uttqlsv.R

class Splash : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)

        val timer = object: Thread() {
            override fun run() {
                try {
                    sleep(1200)
                } catch (e: InterruptedException) {
                    e.printStackTrace()
                } finally {
                    val intent = Intent(applicationContext, Login::class.java)
                    startActivity(intent)
                }
                super.run()
            }
        }

        timer.start()
    }

    override fun onPause() {
        super.onPause()
        finish()
    }
}